hadoop-hdds/rocks-native/pom.xml

```diff
@@ -113,79 +111,80 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>com.googlecode.maven-download-plugin</groupId>
-            <artifactId>download-maven-plugin</artifactId>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
             <executions>
               <execution>
-                <id>rocksdb source download</id>
-                <phase>generate-sources</phase>
-                <goals>
-                  <goal>wget</goal>
-                </goals>
-                <configuration>
-                  <url>https://github.com/facebook/rocksdb/archive/refs/tags/v${rocksdb.version}.tar.gz</url>
-                  <outputFileName>rocksdb-v${rocksdb.version}.tar.gz</outputFileName>
-                  <outputDirectory>${project.build.directory}/rocksdb</outputDirectory>
-                </configuration>
-              </execution>
-              <execution>
-                <id>zlib source download</id>
-                <phase>generate-sources</phase>
-                <goals>
-                  <goal>wget</goal>
-                </goals>
-                <configuration>
-                  <url>${zlib.url}</url>
-                  <outputFileName>zlib-${zlib.version}.tar.gz</outputFileName>
-                  <outputDirectory>${project.build.directory}/zlib</outputDirectory>
-                </configuration>
-              </execution>
-              <execution>
-                <id>bzip2 source download</id>
-                <phase>generate-sources</phase>
+                <id>set-property</id>
+                <phase>initialize</phase>
                 <goals>
-                  <goal>wget</goal>
+                  <goal>java</goal>
                 </goals>
                 <configuration>
-                  <url>${bzip2.url}</url>
-                  <outputFileName>bzip2-v${bzip2.version}.tar.gz</outputFileName>
-                  <outputDirectory>${project.build.directory}/bzip2</outputDirectory>
+                  <mainClass>org.apache.hadoop.hdds.utils.db.managed.JniLibNamePropertyWriter</mainClass>
+                  <arguments>
+                    <argument>${project.build.directory}/propertyFile.txt</argument>
+                  </arguments>
                 </configuration>
               </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>properties-maven-plugin</artifactId>
+            <executions>
               <execution>
-                <id>lz4 source download</id>
-                <phase>generate-sources</phase>
+                <id>read-property-from-file</id>
+                <phase>initialize</phase> <!-- Choose the appropriate phase -->
```

Comment:
Do you intend to change phase? If yes, please do it as part of this change and remove the comment.

Reply:
the value is right. Removed the comment.